src/Components/App/App.js

```diff
@@ -1,21 +1,39 @@
 import React, { Component } from 'react';
+import movieData from '../../movieData'
 import Header from '../Header/Header';
 import Movies from '../Movies/Movies';
-import movieData from '../../movieData'
+import Details from '../Details/Details'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      allMovies: movieData.movies
+      allMovies: movieData.movies,
+      isMovieSelected: false,
+      selectedMovieID: null,
     }
   }
 
+  toggleSeletion = (movieID) => {
+    this.setState({
+      isMovieSelected: !this.state.isMovieSelected, 
```

Comment:
Nice work updating multiple keys in a single setState! This is the pattern we want to use :) good job